src/main/java/com/revature/ams/Booking/BookingController.java

```diff
@@ -52,8 +52,8 @@ private void postBookFlight(Context ctx) {
     }
 
     /**
-     * TODO: DOCUMENT ME
-     * @param ctx
+     * If the member is an admin and memberType is not null, retrieve all bookings.
+     * @param ctx The context from HTTP request. Used to get the member type.
```

Comment:
the description is accurate, however it could be changed so that you describe what it does and in the line after @param you can insert @return and describe what's being returned

Reply:
Method is void so nothing is being returned. Bookings are being shown with ctx.json